src/main.cpp

```diff
@@ -229,19 +232,22 @@ int main(int argc, char *argv[])
             ASYNCMODE_PORT1,
             SensorImuRate / async_output_rate,  // update rate [ms]
             COMMONGROUP_QUATERNION
+			| COMMONGROUP_YAWPITCHROLL
             | COMMONGROUP_ANGULARRATE
             | COMMONGROUP_POSITION
             | COMMONGROUP_ACCEL
             | COMMONGROUP_MAGPRES,
-            TIMEGROUP_NONE,
+            TIMEGROUP_GPSTOW
+            | TIMEGROUP_GPSWEEK
+            | TIMEGROUP_TIMEUTC,
             IMUGROUP_NONE,
             GPSGROUP_NONE,
             ATTITUDEGROUP_YPRU, //<-- returning yaw pitch roll uncertainties
             INSGROUP_INSSTATUS
             | INSGROUP_POSLLA
-            | INSGROUP_POSECEF
-            | INSGROUP_VELBODY
-            | INSGROUP_ACCELECEF,

```

Comment:
Do you want to push back to the public repository? If so we should find a way to have ECEF and NED coexist.